dev/ci/ci-common.sh

```diff
@@ -11,7 +11,8 @@ export NJOBS
 # our docker-opam / Nix setup; we have to do this for all the 3 cases
 # below; would we fix ci-menhir, then we just do this for the first
 # branch [ci case]
-export OCAMLPATH="$PWD/_install_ci/lib:$OCAMLPATH"
+if which cygpath >/dev/null 2>&1; then OCAMLFINDSEP=\;; else OCAMLFINDSEP=:; fi
```

Comment:
Are the ci scripts really expected to be run on cygwin?

Reply:
I don't know, but this does not hurt IMP